scripts/image-names.sh

```diff
@@ -60,23 +61,23 @@ secrel_dockerfile() {
 secrel_docker_context() {
   GRADLE_FOLDER=$(gradle_folder "$1")
   case "$1" in
-    pdfgenerator|assessclaim*) echo "./service-python";;
+    pdfgenerator|featuretoggle|assessclaim*) echo "./service-python";;
     *) echo "$GRADLE_FOLDER/build/docker";;
   esac
 }
 
 secrel_docker_build_args() {
   GRADLE_FOLDER=$(gradle_folder "$1")
   case "$1" in
-    pdfgenerator|assessclaim*) echo "    - SERVICE_SRC_FOLDER=$1/build/docker";;
+    pdfgenerator|featuretoggle|assessclaim*) echo "    - SERVICE_SRC_FOLDER=$1/build/docker";;
     app|console|svc-lighthouse-api) echo "    - JAR_FILE=$1-*.jar
     - ENTRYPOINT_FILE=entrypoint.sh";;
     *) echo "";;
   esac
 }
 
 # These names should match directory names
-IMAGES=( app postgres db-init console svc-lighthouse-api pdfgenerator assessclaimdc7101 assessclaimdc6602 )
```

Comment:
Why is `svc-lighthouse-api` being removed from this list? Possibly a Git mishap?

Reply:
100% git mishap, should be resolved now